go/rules-groups-client.go

```diff
@@ -20,6 +20,79 @@ import (
 	rulesgroups "github.com/coralogix/coralogix-management-sdk/go/internal/coralogix/rules/v1"
 )
 
+const (
+	SeverityConstraint_VALUE_DEBUG_OR_UNSPECIFIED = rulesgroups.SeverityConstraint_VALUE_DEBUG_OR_UNSPECIFIED
```

Comment:
`SeverityConstraint_VALUE_DEBUG_OR_UNSPECIFIED` I changed that to `SeverityConstraintValueDebugOrUnspecified`